packages/reaction-router/common/init.js

```diff
@@ -88,7 +88,7 @@ const getRegistryRouteName = (packageName, registryItem) => {
  */
 ReactionRouter.initPackageRoutes = () => {
   const pkgs = ReactionCore.Collections.Packages.find().fetch();
-  const prefix = ReactionCore.getShopName().toLowerCase(); // todo add shopId
+  const prefix = getSlug(ReactionCore.getShopName().toLowerCase()); // todo add shopId
```

Comment:
well at this point, getSlug is a global function, and currently, this is the only place that the shop name would be useful slugified - I think.